packages/@aws-cdk/aws-logs/lib/metric-filter.ts

```diff
@@ -49,6 +49,7 @@ export class MetricFilter extends Resource {
         metricValue: props.metricValue ?? '1',
         defaultValue: props.defaultValue,
         dimensions: props.dimensions ? Object.entries(props.dimensions).map(([key, value]) => ({ key, value })) : undefined,
+        unit: props.unit ?? Unit.NONE,
```

Comment:
If this were just the following, what would the CloudFormation behavior be?

```suggestion
        unit: props.unit,
```
I'm leaning toward not wanting the conditional of adding `Unit.NONE` to this but I want to make sure I understand what CloudFormation does in this case. Does it assign `Unit.NONE` under the hood?

In general, we don't want to make a change to a template if it isn't changing the behavior of that template. It concerns users when they see changes to their template without changing their code.